src/@types/global.d.ts

```diff
@@ -0,0 +1,9 @@
+// global.d.ts
+import { KcContext } from '../keycloak-theme/login/models/KcContext.js'; // Adjust the path as necessary
```

Comment:
```suggestion
import type { KcContext } from '../keycloak-theme/login/models/KcContext.js'; // Adjust the path as necessary
```
This is surprizing to me to import the types from a js file but I guess in Angular you guys add the extentions....

Reply:
I think js extension could be removed